packages/rulesets/src/spectral/functions/lro-provisioning-state-specified.ts

```diff
@@ -21,9 +21,11 @@ export const provisioningStateSpecified = (pathItem: any, _opts: any, ctx: any)
       }
 
       for (const code of codes) {
-        if (!getProperty(pathItem[verb].responses[code]?.schema, "provisioningState")) {
+        const allProperties = getProperties(pathItem[verb].responses[code]?.schema)
```

Comment:
>getProperties [](http://example.com/codeflow?start=30&length=13)

how are we determining whether this is a long running operation or not? We need to look at the existence of the x-ms-long-running property on the operation. Is that happening in the calling method?

Reply:
yeah!
we narrow down to only to the LRO https://github.com/Azure/azure-openapi-validator/pull/447/commits/c5e3829713a8754ea4a0b1b71a93f0d3792bfdb4#diff-4c1382203db84bcd9df61a5bbf90823d0e1f39a833e8eaa1a5be96ca4a4e9b61R81

Also, I thought from our discussion we planned on implementing this rule for only PUT & PATCH

Reply:
pl feel free to open a separate item for delete if you think it will take longer to incorporate that. That way we know why this might have taken longer to get done.